settings/template.ini

```diff
@@ -72,6 +72,13 @@ Methods[]=ftps
 Methods[]=webdav
 Methods[]=webdavs
 
+# Add support for text which is started from www. without protocol.
+# For example change text "www.ez.no" to link tags.
```

Comment:
We think this setting comment block text should be re-written and clarified. Here is a suggested replacement:

`# Link text urls starting with 'www.' without protocol when using autolink template operator
# For example link text string "www.ez.no". Disabled by default to retain BC.
# Use either enabled or disabled.`